components/inputs/test/input-number.test.js

```diff
@@ -132,17 +132,23 @@ describe('d2l-input-number', () => {
 		it('should update value when language changes', async() => {
 			const elem = await fixture(html`<d2l-input-number label="label" value="2000.3"></d2l-input-number>`);
 			setTimeout(() => documentLocaleSettings.language = 'fr');
-			await oneEvent(elem, 'd2l-localize-behavior-language-changed');
-			await elem.updateComplete;
-			expect(getInnerInputValue(elem)).to.equal('2 000,3');
+			return Promise.all([
+				oneEvent(elem, 'd2l-localize-behavior-language-changed'),
+				elem.updateComplete
+			]).then(() => {
+				expect(getInnerInputValue(elem)).to.equal('2 000,3');
+			});
 		});
 
 		it('should not update empty value when language changes', async() => {
 			const elem = await fixture(html`<d2l-input-number label="label"></d2l-input-number>`);
 			setTimeout(() => documentLocaleSettings.language = 'fr');
-			await oneEvent(elem, 'd2l-localize-behavior-language-changed');
-			await elem.updateComplete;
-			expect(getInnerInputValue(elem)).to.equal('');
+			return Promise.all([
+				oneEvent(elem, 'd2l-localize-behavior-language-changed'),
+				elem.updateComplete
+			]).then(() => {
+				expect(getInnerInputValue(elem)).to.equal('');
+			});
```

Comment:
These tests (and the similar ones in time input) have been flaky since I added them last week. I suspect what's happening is the event and the `updateComplete` promises could be resolving out-of-order, so switching to a `Promise.all` instead which won't care about the sequence.